ote/src/cljs/ote/app/controller/admin_transit_changes.cljs

```diff
@@ -181,6 +181,29 @@
              {:on-success (tuck/send-async! ->LoadHashRecalculations)})
   app)
 
+
+(define-event CSVLoadSuccess [response]
+              {}
```

Comment:
Intention should be two spaces

Reply:
This is how the idea autoformatter formats it, and I couldn't find a way to change it. I think it would be quite bothersome to manually override the autoformatter everytime.